session-2/TaskManagementSystem/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	var (
+		projectStatus = "In Progress"
+		tasksCreated  = 25
+		isCompleted   = false
+	)
+
+	const (
+		low = iota
+		medium
+		high
+		tasksAvailable int = 100
+	)
+
+	fmt.Println(`  Welcome to the Task Management System!
+  Project start date is: 2024-09-18 00:00:00
+  Project: Task Management System`)
+
+	fmt.Println()
+
+	fmt.Printf("  Current project status: %s\n", projectStatus)
```

Comment:
As of the task, "In Progress" remarked as UpperCase string. You can use strings package builtin function
```suggestion
	fmt.Printf("  Current project status: %s\n", strings.ToUpper(projectStatus))
```